### PR TITLE
addpose state + behavior for vision behaviors

### PR DESCRIPTION
**Pull request by supertoto29**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Add the behavior for the vision state to move from a target found. The behavior has a input_key of a Addpose msg. This behavior should not be use to move the sub directly.

## How has this been tested ?
Tested in simulation with the control. The target reached didn't work but the behavior took it in account.

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from sonia_common.msg import MultiAddPose
+
+class add_addpose_to_trajectory(EventState):
+
+    '''
+        Create the pose for the trajectory with a message Addpose.
+        Should only be use for the vision state.
+
+        ># pose             AddPose         Input pose
+        ># input_traj       MultiAddPose    Input trajectory
```
Comment (Kam0ss) [inline sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py]: Inverted in input keys, will cause confusion

File: sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    def execute(self, userdata):
+        traj = userdata.input_traj
+        pose = userdata.pose
+        new_traj = MultiAddPose()
+        if not traj.pose:
+            Logger.log('First position of the trajectory', Logger.REPORT_HINT)
+        else:
+            Logger.log('Adding a pose to the trajectory', Logger.REPORT_HINT)            
+            new_traj.pose = list(traj.pose)
```
Comment (Kam0ss) [inline sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py:35]: Interpolation method not used.

**Review by Kam0ss — CHANGES_REQUESTED** [review]
Changes to do + behavior name is not really explicit

File: sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py (hunk 1 above)
Comment (supertoto29) [reply sonia_navigation_states/src/sonia_navigation_states/add_addpose_to_trajectory.py]: I don't understand. What do you by inverted?

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**